dali/pipeline/data/tensor.h

```diff
@@ -136,6 +136,18 @@ class Tensor : public Buffer<Backend> {
     shape_ = shape;
   }
 
+  /**
+   * @brief Tensor is always backed by contiguous buffer
+   */
+  bool IsContiguous() {
+    return true;
+  }
+
+  /**
+   * @brief Tensor is always backed by contiguous buffer - No Op
+   */
+  void SetContiguous(bool) {}
```

Comment:
Maybe it could throw or fail anyhow, than just silently do nothing?

Reply:
I guess, I can add a DALI_ENFORCE for the arg to be true.

Reply:
I think it might do.

Reply:
Done